doc/tutorials/run_all_tests_with_docker.md

```diff
@@ -67,7 +66,7 @@ So from your root project folder run the following:
 docker run -it --rm \
 -v "$PWD:/home/jenkins/workspace" \
 -w /home/jenkins/workspace \
-<image_name>:<tag_name>
+hub-public.libelektra.org/<image_name>:<tag_name>
```

Comment:
Here the user discussed in the issue #2781 is still missing.

Reply:
The proposed solution introduces other problems. If running with the current user, not just 1 test fails, but around 20 of them.

Reply:
Seems like the problem can only be fixed if you build the docker container correctly. See scripts/docker/README.md